sockeye/inference.py

```diff
@@ -1144,10 +1147,10 @@ def __init__(self,
         utils.check_condition(self.beam_size >= nbest_size,
                               'Nbest size must be smaller or equal to beam size.')
         self.batch_size = self.models[0].batch_size
-        # skip softmax for a single model, but not for an ensemble
-        self.skip_softmax = self.models[0].skip_softmax
-        if self.skip_softmax:
-            utils.check_condition(len(self.models) == 1 and self.beam_size == 1, "Skipping softmax cannot be enabled for several models, or a beam size > 1.")
+
+        if self.models[0].skip_softmax:
```

Comment:
That check doesn't seem to be complete. If there is more than one model we now would need to make sure that none of the models have `skip_softmax` set, right? (The check was already like this before the PR, but I'm just wondering).

Reply:
I think this requirement is implicitly given by the checks in `load_models()`, but I can extend the check here.